libraries/plugins/vfio/opae_vfio.c

```diff
@@ -1590,6 +1598,63 @@ fpga_result __VFIO_API__ vfio_fpgaGetIOAddress(fpga_handle handle,
 	return FPGA_OK;
 }
 
+fpga_result __VFIO_API__ vfio_fpgaBindSVA(fpga_handle handle, uint32_t *pasid)
+{
+	vfio_handle *h;
+	char path[PATH_MAX];
+	int fd;
+	int err;
+	fpga_result res = FPGA_OK;
+
+	h = handle_check_and_lock(handle);
+	ASSERT_NOT_NULL(h);
+	ASSERT_NOT_NULL(pasid);
```

Comment:
If we want to allow pasid to be NULL, this will prevent that. Better to add a NULL check at 1621 and 1642.